extensions/roc-plugin-style-sass/src/index.js

```diff
@@ -32,11 +32,16 @@ export const roc = {
         extension: 'roc-plugin-style-css',
         hook: 'add-style',
         description: 'Adds Sass support to Webpack.',
-        action: ({ context: { config: { settings } } }) => () => () => ({
-            extensions: ['sass', 'scss'],
-            loaders: `${require.resolve('sass-loader')}?${settings.build.sass.useBourbon ?
-                `includePaths[]=${BOURBON_PATH}` : ''
-            }`,
-        }),
+        action: ({ context: { config: { settings } } }) => () => () => {
+            const params = [
+                settings.build.sass.useBourbon ? `includePaths[]=${BOURBON_PATH}` : '',
+                settings.build.style.sourceMap ? 'sourceMap' : '',
+            ].filter(v => v !== '');
```

Comment:
I agree with the empty string being falsy. I still think the filter check is needed, because:

```
% node
> ['foo', '', 'bar'].join('&')
'foo&&bar'
> ['foo', '', 'bar'].filter(v => v !== '').join('&')
'foo&bar'
```